gradle/verification-metadata.xml

```diff
@@ -1227,6 +1227,11 @@
             <pgp value="BE685132AFD2740D9095F9040CC0B712FEE75827"/>
          </artifact>
       </component>
+      <component group="org.beanshell" name="bsh" version="2.0b4">
+         <artifact name="bsh-2.0b4.jar">
+            <sha256 value="91395c07885839a8c6986d5b7c577cd9bacf01bf129c89141f35e8ea858427b6" origin="Generated by Gradle" reason="Artifact is not signed"/>
```

Comment:
This must be changed, see top of the file

Reply:
@cobexer - Thanks for pointing this out.  I wasn't expecting to have to change anything here in this PR, so I was hoping someone would comment on this.

When I read the comment, it says run `./gradlew -i -write-verification-metadata sha256,pgp -export-keys resolveAllDependencies`.  If I delete my addition to this file, running that command recreates it as is.

[Looking at this artifact in Maven Central](https://repo1.maven.org/maven2/org/beanshell/bsh/2.0b4/) I don't see an `.asc` file, so I'm not sure what more to do here.

Any ideas?

Reply:
Added notes about capabilities conflict after Louis shed some light on what was happening here.